src/ORM/Association/BelongsToMany.php

```diff
@@ -115,6 +115,13 @@ class BelongsToMany extends Association
     protected $_through;
 
     /**
+     * Valid strategies for this type of association
+     *
+     * @var array
+     */
+    protected $_validStrategies = [parent::STRATEGY_SELECT, parent::STRATEGY_SUBQUERY];
```

Comment:
I went with `parent` since that's what's currently in the classes for the default `$_strategy`..
``protected $_strategy = parent::STRATEGY_SELECT;``

Let me know if I should change all the references in the classes to the strategy types to `self`.